modules/ossm-about-collecting-ossm-data.adoc

```diff
@@ -31,3 +31,12 @@ $ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-mus
 ----
 $ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8:2.4 gather <namespace>
 ----
+
+This creates a local directory that contains the following: 
```

Comment:
```suggestion
This creates a local directory that contains the following items:
```
IBM Style [Word Usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage) provides this guidance about the term _following_: "To introduce a list, procedure, or subsequent items, always use _following_ as an adjective before a noun. For example, do not write “Complete the following” or ”The functions include the following”. Instead, write “Complete the following steps” or “The adapter has the following functions”."